crypto/bn/bn_div.c

```diff
@@ -446,8 +446,10 @@ int bn_div_fixed_top(BIGNUM *dv, BIGNUM *rm, const BIGNUM *num,
     snum->neg = num_neg;
     snum->top = div_n;
     snum->flags |= BN_FLG_FIXED_TOP;
-    if (rm != NULL)
-        bn_rshift_fixed_top(rm, snum, norm_shift);
+
+    if (rm != NULL && bn_rshift_fixed_top(rm, snum, norm_shift) == 0)
```

Comment:
`!bn_rshift_fixed_top()` is okay here -- it's returning a 0/1 value.

A better habit, given the number of times it's tripped us up, might be `<= 0`.

No need for either change, this is good as it is.